### DISCUSSION TITLE
Filter repositories by multiple traits

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44029] [Repositories]
### Select Topic Area

Product Feedback

### Body

_Admittedly, I can't do this in a browser either, but, maybe you can get it fixed in both places :) ..._

I'd like to filter for Repositories that are `Private` + `Template` or `Public`+`Template` or `Template`+`Fork` or `Template`+`Source` or `Private`+`Template`+`Source`.

The current behavior is limiting and it's worse on mobile because at least in browser even on mobile, I can see the traits in the list:

https://github.com/jsoref?tab=repositories&q=&type=template&language=&sort=
![image](https://user-images.githubusercontent.com/2119212/211933689-9e278bec-aa85-4eaa-8b8d-bab6b2eb9a32.png)

**Comment by stevepopovich** [comment]
Moving to `Repositories` category as that is where this would need to be added for us to implement on Mobile. Thanks!
